dataclients/kubernetes/definitions/ingressv1.go

```diff
@@ -3,10 +3,17 @@ package definitions
 import (
 	"encoding/json"
 	"errors"
-	"fmt"
 	"strconv"
 )
 
+const (
+	SkipperfilterAnnotationKey    = "zalando.org/skipper-filter"
```

Comment:
We should highlight that these are Ingress annotations and I think we do not need to have "Key" suffix, so:
```
IngressFilterAnnotation
IngressPredicateAnnotation
IngressRoutesAnnotation
```